benches/functions/stark.rs

```diff
@@ -0,0 +1,256 @@
+            blowup_factor: 2,
+            fri_number_of_queries: 1,
+            coset_offset: 3,
+        },
+        trace_length,
+        trace_columns: trace.len(),
+        transition_degrees: vec![1],
+        transition_exemptions: vec![2],
+        transition_offsets: vec![0, 1, 2],
+        num_transition_constraints: 1,
+    };
+
+    let fibonacci_air = simple_fibonacci::FibonacciAIR::from(context);
+
+    let result = prove(&trace, &fibonacci_air, &mut ()).unwrap();
+    verify(&result, &fibonacci_air, &());
+}
+
+pub fn prove_fib_2_cols() {
+    let trace_columns =
+        fibonacci_2_columns::fibonacci_trace_2_columns([FE::from(1), FE::from(1)], 16);
+
+    let context = AirContext {
+        options: ProofOptions {
+            blowup_factor: 2,
+            fri_number_of_queries: 1,
+            coset_offset: 3,
+        },
+        trace_length: trace_columns.len(),
+        transition_degrees: vec![1, 1],
+        transition_exemptions: vec![1, 1],
+        transition_offsets: vec![0, 1],
+        num_transition_constraints: 2,
+        trace_columns: 2,
+    };
+
+    let fibonacci_air = fibonacci_2_columns::Fibonacci2ColsAIR::from(context);
+
+    let result = prove(&trace_columns, &fibonacci_air, &mut ()).unwrap();
+    verify(&result, &fibonacci_air, &());
+}
+
+#[allow(dead_code)]
+pub fn prove_fib17() {
+    let trace = simple_fibonacci::fibonacci_trace([FE17::from(1), FE17::from(1)], 4);
+
+    let context = AirContext {
+        options: ProofOptions {
+            blowup_factor: 2,
+            fri_number_of_queries: 1,
+            coset_offset: 3,
+        },
+        trace_length: trace.len(),
+        trace_columns: trace[0].len(),
+        transition_degrees: vec![1],
+        transition_exemptions: vec![2],
+        transition_offsets: vec![0, 1, 2],
+        num_transition_constraints: 1,
+    };
+
+    let fibonacci_air = fibonacci_f17::Fibonacci17AIR::from(context);
+
+    let result = prove(&trace, &fibonacci_air, &mut ()).unwrap();
+    verify(&result, &fibonacci_air, &());
+}
+
+#[allow(dead_code)]
+pub fn prove_quadratic() {
+    let trace = quadratic_air::quadratic_trace(FE::from(3), 16);
+
+    let context = AirContext {
+        options: ProofOptions {
+            blowup_factor: 2,
+            fri_number_of_queries: 1,
+            coset_offset: 3,
+        },
+        trace_length: trace.len(),
+        trace_columns: trace.len(),
+        transition_degrees: vec![2],
+        transition_exemptions: vec![1],
+        transition_offsets: vec![0, 1],
+        num_transition_constraints: 1,
+    };
+
+    let quadratic_air = quadratic_air::QuadraticAIR::from(context);
+
+    let result = prove(&trace, &quadratic_air, &mut ()).unwrap();
+    verify(&result, &quadratic_air, &());
+}
+
+// We added an attribute to disable the `dead_code` lint because clippy doesn't take into account
+// functions used by criterion.
+
+#[allow(dead_code)]
+pub fn prove_cairo_fibonacci_5() {
```

Comment:
If the returned value has a big `Drop` (the time it takes is comparable to proof generation), we should return the proof and use `bench.iter_with_large_drop` to call this.